app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -33,7 +32,7 @@ def creation_fields
 
     def generate_attributes
       creation_fields.each do |column|
-        @params[column] = @service_offering.send(column) if @service_offering.send(column).present?
+        @params[column] ||= @service_offering.send(column) if @service_offering.send(column).present?
```

Comment:
Why should this be `||=`? The only time it would already be set would be if the Topology response had duplicates which isn't possible.

Reply:
nevermind, after pulling this pr down and inspecting it it is for when the user defines properties on creation, so they don't get overwritten by the topology properties.